nucypher/blockchain/eth/networks.py

```diff
@@ -30,18 +30,22 @@ class NetworksInventory:  # TODO: See #1564
     UNKNOWN = 'unknown'  # TODO: Is there a better way to signal an unknown network?
     DEFAULT = MAINNET
 
-    __to_ethereum_chain_id = {
+    __to_ethereum_chain_id_eth = {
         MAINNET: 1,      # Ethereum Mainnet
         ORYX: 3,         # Ropsten
         IBEX: 4,         # Rinkeby
         LYNX: 5,         # Goerli
-
+    }
+    __to_ethereum_chain_id_poly = {
         # TODO: Use naming scheme?
         POLYGON: 137,    # Polygon Mainnet
         MUMBAI: 80001,   # Polygon Testnet (Mumbai)
     }
 
-    NETWORKS = tuple(__to_ethereum_chain_id.keys())
+    ETH_NETWORKS = tuple(__to_ethereum_chain_id_eth.keys())
+    POLY_NETWORKS = tuple(__to_ethereum_chain_id_poly.keys())
+
+    NETWORKS = ETH_NETWORKS + POLY_NETWORKS
```

Comment:
Is this used anymore anywhere? If it still is, I would look into whether its usage can simply just be changed to ETH_NETWOKRS...

Reply:
it's used for verification in a few places, for example in `nucypher/blockchain/eth/registry.py`

```python
class CanonicalRegistrySource(ABC):

    logger = Logger('RegistrySource')

    name = NotImplementedError
    is_primary = NotImplementedError

    def __init__(self, network: str, registry_name: str, *args, **kwargs):
        if network not in NetworksInventory.NETWORKS:
            raise ValueError(f"{self.__class__.__name__} not available for network '{network}'. "
                             f"Valid options are: {list(NetworksInventory.NETWORKS)}")
        self.network = network
        self.registry_name = registry_name
```